lib/sdam/server_description.js

```diff
@@ -66,6 +69,28 @@ class ServerDescription {
       ismaster
     );
 
+    // https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#serverdescription
+    // set up defaults / required properties
+    this.address = address;
+    this.error = null;
+    this.roundTripTime = null;
```

Comment:
Some of these get immediately re-assigned a few lines down, doesn't look intended.

Reply:
I guess I left it like that because I wanted to make it clear that these were the defaults as specified in the spec and that I had to add them in order to get tsc to know that they are indeed properties of the class. But it is indeed clunky and potentially confusing so I'll combine and reduce